core/templates/components/oppia-angular-root.component.ts

```diff
@@ -123,6 +123,9 @@ const componentMap = {
     component_class: NoninteractiveVideo,
   }
 };
+
+let areRteElementsInitialized = false;
```

Comment:
```suggestion
let rteElementsAreInitialized = false;
```
(Avoid having boolean variable nmae with verb prefix, generally we have such name for functions.)

Reply:
WHy is this defined in the module scope? (And not in the component scope?)

Reply:
We want rteElements to be only created once per page. Either it has to be static or defined in the module scope.

What are the advantages of defining it as a static member instead of defining in module scope?

Reply:
Made it static in the component scope.